### PR TITLE
Fixes #513, #512: show/hide show de novo on reference genome removed

### LINKED ISSUE #513: In Alignments view, ReferenceGenome missing 'New' button

### PR DESCRIPTION
**Pull request by woodymit**
This should fix Nili's two issues.

### CODE REVIEW

File: genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py [hunk 1]
```diff
@@ -10,13 +10,14 @@ class Migration(SchemaMigration):
     def forwards(self, orm):
         # Adding field 'ReferenceGenome.metadata'
         db.add_column(u'main_referencegenome', 'metadata',
-                      self.gf('main.custom_fields.PostgresJsonField')(default='{"is_from_de_novo_assembly": false}'),
+                      self.gf('main.custom_fields.PostgresJsonField')(default='{}'),
```
Comment (glebkuznetsov) [inline genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py]: These changes to auto-generated migration are a bit sketch. Are you sure they are correct?

Comment (woodymit) [inline genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py]: Yup I tested backward and forward migrations

File: genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py [hunk 2]
```diff
@@ -10,13 +10,14 @@ class Migration(SchemaMigration):
...
                       keep_default=False)
 
 
     def backwards(self, orm):
         # Deleting field 'ReferenceGenome.metadata'
         db.delete_column(u'main_referencegenome', 'metadata')
+        # db.delete_column(u'main_chromosome', 'seqrecord_id')
```
Comment (glebkuznetsov) [inline genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py]: Get rid of this line.

**Comment by glebkuznetsov** [comment]
Please remove that extra comment you added. Otherwise everything looks good and I'll merge when that's done. Thanks!

**Pull request merged**